projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -721,7 +721,15 @@ public static function enqueue_block_editor_assets() {
 					apply_filters( 'jetpack_subscriptions_newsletter_feature_enabled', false )
 					&& class_exists( '\Jetpack_Memberships' )
 				),
-				'is_newsletter_configured'      => self::is_newsletter_configured(),
+				/**
+				 * Show the Paid Newsletter access panel selector in every post sidebar.
+				 *
+				 * @module subscriptions
+				 * @since 12.0
```

Comment:
Just a nitpick, but this doesn't look correct. You can use `@since $$next-version##` here to make the version automatically get substituted on a public release.